searchlib/src/vespa/searchcommon/attribute/iattributevector.h

```diff
@@ -24,6 +25,7 @@ namespace search::attribute {
 
 class IMultiValueAttribute;
 class ISearchContext;
+class ISortBlobWriter;
```

Comment:
Not needed, class is already defined due to including `default_sort_blob_writer.h`.

Reply:
Resolved offline.